hooks/ti.dynamiclib.js

```diff
@@ -39,7 +39,8 @@ exports.init = function (logger, config, cli, appc) {
 			var scriptPath = '$BUILT_PRODUCTS_DIR/$FRAMEWORKS_FOLDER_PATH/PSPDFKit.framework/strip-framework.sh'; // Or set to null if not required
 			var frameworkPaths = [
         			// Replace with the path of your embedded framework. Make sure the path is relative to `build/iphone`
-				'../../Resources/iphone/PSPDFKit.framework'
+				'../../Resources/iphone/PSPDFKit.framework',
+				'../../Resources/iphone/PSPDFKitUI.framework'
```

Comment:
Btw, Ti SDK 6.2.0+ supports dynamic frameworks built-in, so now manual library is required. You might want to guard that in the dynamic-lib hook or even bump the minimum SDK version.

Reply:
We'll just bump the version - I don't see a benefit in supporting older Ti SDKs.

Thanks!

Reply:
> Btw, Ti SDK 6.2.0+ supports dynamic frameworks built-in, so no manual linking is required.

@hansemannn are you suggesting to remove the lines below from our `hooks/ti.dynamiclib.js` and bump the Titanium min SDK version so we can use dynamic frameworks instead of manual linking?

```javascript
'../../Resources/iphone/PSPDFKit.framework',
'../../Resources/iphone/PSPDFKitUI.framework'
```

If so, then I'm not sure I understand the changes that are required in 6.2.0+.

I did see [TIMOB-20557](https://jira.appcelerator.org/browse/TIMOB-20557) and https://github.com/appcelerator-modules/hook-swift-frameworks, and it didn't seem to work.

Thank you!

Reply:
Hey @radazzouz. Unless `PSPDFKit` or `PSPDFKitUI` is a Swift-based framework, you won't need the Swift-hook. 

6.2.0 supports dynamic / embedded frameworks, which implements the ti.dynamiclib hook into the core. If you see problems while removing that hook, you should definitely ping us to address it. 

And if one of the frameworks is Swift-based, you would need the Swift-hook as well, which is not in the core so far. Using it requires you to set the Swift version as Swift does not have ABI stability so far. 

Let me know if you need more insights!